int32 time

---

multi_contact_planning/CartesioPlannerGoalStatus status
sensor_msgs/JointState sampled_goal
